app/src/main/java/com/portfolio/tasky/entry/viewModels/LoginViewModel.kt

```diff
@@ -34,5 +40,8 @@ class LoginViewModel @Inject constructor(
     fun areFieldsValid() {
         mutableFieldsValid.value =  mutableEmail.value == true && mutablePassword.value == true
     }
+    fun makeLoginCall(authenticationModel: AuthenticationRequest) : LiveData<AuthenticationResponse?>{
+        return entryRepository.doLogin(taskyCallStatus, authenticationModel)
```

Comment:
When using coroutines, you'll have to use 
```
viewModelScope.launch {
entryRepository.doLogin...
}
```

Reply:
Current implementation not going with this. will find a better way to implement coroutine with existing functionality.

Reply:
Done